### PR TITLE
Frontend assignment

### PR DESCRIPTION
**Pull request by ricardoferrolho**

### CODE REVIEW

File: front-end.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Head into `front-end/q4/server` and install the server dependencies via:
+
+```bash
+yarn install
+```
+
+Run the server using:
+
+```bash
+yarn server
```
Comment (alecgibson) [inline front-end.md]: (Or `npm install && npm run server`)

File: front-end.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+The server should be running on port `3000`.
+
+#### Available routes
+
+#### http://localhost:3000/books
+
+Returns a list of books, with their info.
+
+#### http://localhost:3000/books/*slug*
```
Comment (alecgibson) [inline front-end.md]: (The `*slug*` displays a bit weird on the Github-rendered markdown)

File: front-end/q3/q3.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Front end Engineer Challenge Q3</title>
+  </head>
+  <body>
+    <div class="top-actions">
+      <div class="container">
+        <a class="top-logo">Logo</a>
```
Comment (alecgibson) [inline front-end/q3/q3.html]: (Don't know if this was meant to be an actual logo? If so, the screenshot will also need updating)

File: front-end/q4/server/yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,394 @@
+Arguments:
```
Comment (alecgibson) [inline front-end/q4/server/yarn-error.log]: Should probably `.gitignore` this file

File: node-fullstack.md [hunk 5]
```diff
@@ -60,7 +60,7 @@ Add test coverage as you see fit.
 
 The project should be responsible for managing all the required dependencies and should run just by using `npm install` and `npm start`.
 
-
+<!--  -->
```
Comment (alecgibson) [inline node-fullstack.md]: ?

File: front-end/q3/q3.html (hunk 3 above)
Comment (ricardoferrolho) [reply front-end/q3/q3.html]: Ah, no. It's just a placeholder, no need to create a logo just for the challenge.

File: front-end.md (hunk 1 above)
Comment (ricardoferrolho) [reply front-end.md]: Added.

File: front-end.md (hunk 2 above)
Comment (ricardoferrolho) [reply front-end.md]: Updated.

**Comment by sebastiansito** [comment]
Nice one, I did a review but honestly was more like looking for typos 😄 👌

**Pull request merged**